lib/workerholic/statistics_api.rb

```diff
@@ -46,11 +54,20 @@ def self.get_jobs_for_class(job_class)
         JobSerializer.deserialize_stats(serialized_job)
       end
 
-      deserialized_stats
+      deserialized_stats << deserialized_stats.size
+    end
+
+    def self.jobs_per_class(job_class)
+      clean_class_name = job_class.split(':').last
+      [clean_class_name, self.storage.list_length(job_class)]
     end
 
     def self.storage
-      storage ||= Storage::RedisWrapper.new
+      Storage::RedisWrapper.new
```

Comment:
you should use a class instance variable @storage so you don't keep creating instances of Storage::RedisWrapper.new